distribution/docs/src/main/resources/content/_configuring/changing-hostname.adoc

```diff
@@ -7,7 +7,7 @@
 
 == {title}
 
-By default Solr, STS server, STS client and the rest of the services use the system property `org.codice.ddf.system.hostname` which is defaulted to 'localhost' and not to the fully qualified domain name of the ${branding} instance.
+By default STS server, STS client and the rest of the services use the system property `org.codice.ddf.system.hostname` which is defaulted to 'localhost' and not to the fully qualified domain name of the ${branding} instance.
```

Comment:
Change grammar here with "By default STS server,...". Maybe add "the" before "STS" to make it read better.